### DISCUSSION TITLE
APISIX starts but then fails

### DISCUSSION THREAD

**Discussion by Elf36** [apache/apisix#7983] [General]
/usr/local/openresty/luajit/bin/luajit /usr/local/apisix/apisix/cli/apisix.lua start
APISIX is running...
[apisix@apisix ~]$ systemctl status apisix
● apisix.service - apisix
   Loaded: loaded (/usr/lib/systemd/system/apisix.service; enabled; vendor preset: disabled)
   Active: failed (Result: start-limit) since Fri 2022-09-23 11:10:37 BST; 43min ago

Centos 7 clean install rpm installation of offline rpm - limits raised at system level

Etcd installed as per documentation

**Comment by tzssangglass** [comment]
ref: https://github.com/apache/apisix/issues/6401#issuecomment-1047016902
